### PR TITLE
Display transaction history in pages on nuklear

### LINKED ISSUE #314: nuklear, crash on large history list
nuklear history view ends up going into high cpu use due to it trying to load a very large history. App has to be force quit. There needs to be some type of safety limiting mechanism here. The entire history should not be loaded. We likely only want to load items as the user scrolls if nuklear can do this.

### PR DESCRIPTION
**Pull request by yaowork**
This fetches and displays transaction history in pages on the history page of the nuklear interface. In navigating using "previous" and "next" buttons, this PR makes sure that already fetched transaction pages are just re-rendered instead of  fetched again from the walletrpc server. 

It should be noted that the nucular/nuklear project does not provide us with a scroll listener, so this PR makes use of "next" and "previous" buttons to navigate through pages instead of the infinite scrolling suggested by #314   

Fixes #314 

Screenshot from 2019-04-22 18-01-23
Screenshot from 2019-04-22 18-01-44

### CODE REVIEW

**Comment by raedah** [comment]
Currently a random number of items are being returned on each page. This should be fixed so the number of items returned are consistent. This is also an obvious behavior that should have been noted in the PR description when it was submitted.

**Review by ademuanthony — CHANGES_REQUESTED** [review]
Is infinite scrolling not possible as requested in the #314 being resolved? Please update the description to reflect this.
Also https://github.com/raedahgroup/godcr/pull/328#issuecomment-485585342

**Comment by yaowork** [comment]
The PR has been updated to display a consistent number of transactions per page.

**Review by ademuanthony — APPROVED** [review]

File: nuklear/pagehandlers/sync.go [hunk 1]
```diff
@@ -91,6 +91,9 @@ func (s *SyncHandler) Render(window *nucular.Window, changePage func(*nucular.Wi
 		return
 	}
 
+	changePage(window, "overview")
+	return
```
Comment (itswisdomagain) [inline nuklear/pagehandlers/sync.go]: Forgot to remove this.

**Review by itswisdomagain — CHANGES_REQUESTED** [review]

**Review by itswisdomagain — APPROVED** [review]

**Pull request merged**